src/askui_runner/modules/core/infrastructure/runner/askui.py

```diff
@@ -74,11 +75,16 @@ def create_jinja_env(self, dir_path: str) -> jinja2.Environment:
 
     def render_templates(self, dir_path: str) -> None:
         jinja_env = self.create_jinja_env(dir_path=dir_path)
-        templates = [template for template in jinja_env.list_templates() if template.endswith(".jinja")]
+        templates = [
+            template for template in jinja_env.list_templates(extensions=[AskUiJestRunnerService._TEMPLATE_EXTENSION])
+        ]
+        if len(templates) == 0:
+            raise Exception(f"No templates found to render in project template directory {self.project_dir}")
         for template in templates:
+            template_name_without_extension = template[:-(len(AskUiJestRunnerService._TEMPLATE_EXTENSION) + 1)] # +1 for the dot
```

Comment:
With the changes above, this becomes
```suggestion
            template_name_without_extension = template[:-len(AskUiJestRunnerService._TEMPLATE_EXTENSION)] 
```

Reply:
it's either we add +1 here or we remove the dot from template_extension inside the jinja_env.list_templates

Reply:
Okay, then I take everything back. From the usage examples of `list_templates` I thought that it would also allow passing the extension with the full stop.